test/test_ngram.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import os
+import kenlm
+from math.isclose
+
+from espnet.nets.pytorch_backend.lm.ngram import NgramFullScorer
+from espnet.nets.pytorch_backend.lm.ngram import NgramPartScorer
+
+test_sens = ["I like apple", "you love coffee", "I love you", "I love coffee"]
+@pytest.mark.parameterize(test_sens)
+def test_ngram_build(test_sens):
+    os.system(f"lmplz --discount_fallback -o 3 {'\n'.join(test_sens)} >tmp.arpa")
+    os.system("build_binary tmp.arpa tmp.bin")
```

Comment:
This test is OK. However, I prefer storing `test/ngram.arpa` and `test/ngram.bin` rather than generating them in test because this test requires `kenlm` installation.

Reply:
I thinks store .arpa file only is enough